pkg/storage/utils/decomposedfs/upload.go

```diff
@@ -358,33 +360,6 @@ func (fs *Decomposedfs) GetUpload(ctx context.Context, id string) (tusd.Upload,
 	}, nil
 }
 
-// lookupNode looks up nodes by path.
-// This method can also handle lookups for paths which contain chunking information.
-func (fs *Decomposedfs) lookupNode(ctx context.Context, path string) (*node.Node, error) {
```

Comment:
this was just added by @C0rby where did it go?

Reply:
wops, good catch, lost in translation